project/ROADMAP.md

```diff
@@ -0,0 +1,5 @@
+Galaxy Project Roadmap
```

Comment:
@martenson Not sure what you mean, can you say a little more about how it'd look?  As mentioned in the other discussion, I mainly just wanted it as a permanent historical record of the primary roadmap that was portable outside of github.